<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/syncopa.png</normaloff>:/icons/syncopa.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenBlueConfig"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionUpdateOnIdle"/>
    <addaction name="actionShowFPSOnIdleUpdate"/>
    <addaction name="actionBackgroundColor"/>
    <addaction name="actionNetworkConnection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenBlueConfig"/>
   <addaction name="separator"/>
   <addaction name="actionBackgroundColor"/>
   <addaction name="separator"/>
   <addaction name="actionNetworkConnection"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionUpdateOnIdle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update on idle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/colorpicker.png</normaloff>:/icons/colorpicker.png</iconset>
   </property>
   <property name="text">
    <string>Background color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionShowFPSOnIdleUpdate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show FPS on idle update</string>
   </property>
   <property name="toolTip">
    <string>Show FPS on idle update</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenBlueConfig">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/folder-bc.svg</normaloff>:/icons/folder-bc.svg</iconset>
   </property>
   <property name="text">
    <string>Open BlueConfig...</string>
   </property>
   <property name="toolTip">
    <string>Open BlueConfig</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export scene and synapses</string>
   </property>
   <property name="toolTip">
    <string>Export scene and synapses</string>
   </property>
  </action>
  <action name="actionNetworkConnection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/connect.svg</normaloff>:/icons/connect.svg</iconset>
   </property>
   <property name="text">
    <string>Network Connection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
